web-app/helpers/certificate-designs.ts

```diff
@@ -87,45 +98,39 @@ export async function populateDeveloperCert(canvas: Canvas, details: any) {
   switch (programCode) {
```

Comment:
I don't know what TR101 and all of those other program codes came from. But if they are not official and if we need to define a list of official program codes, what I would do is propose a list of reasonable ones. And add that list as documentation in the Readme somewhere.

Maybe just codes like Developer, Analyst, Designer, etc.  Those are better in my opinion than abbreviations like NCD since NCD gets confusing since it could represent Developer or Designer. Nobody would know from looking at it.

Rename each background image in https://github.com/NEAR-Edu/near-certification-tools/tree/feature/update-certificate-designs/web-app/public/certificate-backgrounds to remove the NCD etc prefixes and instead have the new program codes there instead.

Then, remove this entire switch section. It can be handled with just 1 line, where the name of the image is composed dynamically via the program code.

Reply:
done